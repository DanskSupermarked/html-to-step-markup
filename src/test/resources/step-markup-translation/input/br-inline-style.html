<p><strong style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);">Ultraslankt LED-TV med Ambilight</strong><br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <span style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); display: inline !important; float: none;">Nu kan du få en TV-oplevelse i særklasse med et 3D Smart LED-TV fra Philips' 6008-serie. Lad dig ikke narre af det slanke design - det er fyldt med funktioner som f.eks. Skype, 3D, 2-sidet Ambilight og meget mere.</span><br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <strong style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);">En TV-oplevelse uden rammer</strong><br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <span style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); display: inline !important; float: none;">Det rammefrie design minimerer skærmrammen for at skabe et mere raffineret udseende, mens et perfekt finish sørger for et TV, hvor udseendet er lige så fantastisk som ydeevnen.</span><br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <strong style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);">En verden af onlineindhold lige ved fingerspidserne</strong><br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <span style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); display: inline !important; float: none;">Med Skype bliver det nemt at foretage stemme- og videoopkald på dit TV. Og integreret Wi-Fi giver dig nem adgang til onlineforbindelse til resten af verden, mens Smart TV åbner dørene til en verden af onlineunderholdning.</span><br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <strong style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);"><br /> Få kontrol over, hvordan du deler og optager på den store skærm</strong><br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <span style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); display: inline !important; float: none;">Du får fjernbetjening med tastatur, som gør det nemt at indtaste tekst og en MyRemote-app, der nemt lader dig betjene dit TV med din telefon eller tablet. Med Wi-Fi Miracast kan du afspejl dine enheder på dit TV, mens SimplyShare lader dig nyde fotos, musik og film på TV'et.</span><br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <strong style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);">En rigere underholdningsoplevelse</strong><br style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255);" /> <span style="color: rgb(0, 0, 0); font-family: Arial, Verdana, sans-serif; font-size: 12px; font-style: normal; font-variant: normal; font-weight: normal; letter-spacing: normal; line-height: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); display: inline !important; float: none;">En 2-sidet Ambilight XL forstærker din TV-oplevelse og Pixel Precise HD giver skarpere detaljer ved bevægelse.</span></p> <p><br /> Med TV'et medfølger 2 mdr.’s Boxer  Max inkl. halvårlig kortafgift og Boxer TV-Modul – værdi 1.296 kr. ( 2 x 299 kr. for Boxer Max + 399 kr. i halvårlig kortafgift + 299 kr. for modul).<br /> <br /> Fødselsdagstilbudet med TV og Boxer gælder så længe lager haves.</p> <p> </p>
