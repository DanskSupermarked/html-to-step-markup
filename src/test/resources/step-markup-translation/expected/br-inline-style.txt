<bold>Ultraslankt LED-TV med Ambilight</bold>
Nu kan du få en TV-oplevelse i særklasse med et 3D Smart LED-TV fra Philips' 6008-serie. Lad dig ikke narre af det slanke design - det er fyldt med funktioner som f.eks. Skype, 3D, 2-sidet Ambilight og meget mere.

<bold>En TV-oplevelse uden rammer</bold>
Det rammefrie design minimerer skærmrammen for at skabe et mere raffineret udseende, mens et perfekt finish sørger for et TV, hvor udseendet er lige så fantastisk som ydeevnen.

<bold>En verden af onlineindhold lige ved fingerspidserne</bold>
Med Skype bliver det nemt at foretage stemme- og videoopkald på dit TV. Og integreret Wi-Fi giver dig nem adgang til onlineforbindelse til resten af verden, mens Smart TV åbner dørene til en verden af onlineunderholdning.
<bold>
Få kontrol over, hvordan du deler og optager på den store skærm</bold>
Du får fjernbetjening med tastatur, som gør det nemt at indtaste tekst og en MyRemote-app, der nemt lader dig betjene dit TV med din telefon eller tablet. Med Wi-Fi Miracast kan du afspejl dine enheder på dit TV, mens SimplyShare lader dig nyde fotos, musik og film på TV'et.

<bold>En rigere underholdningsoplevelse</bold>
En 2-sidet Ambilight XL forstærker din TV-oplevelse og Pixel Precise HD giver skarpere detaljer ved bevægelse.

Med TV'et medfølger 2 mdr.’s Boxer Max inkl. halvårlig kortafgift og Boxer TV-Modul – værdi 1.296 kr. ( 2 x 299 kr. for Boxer Max + 399 kr. i halvårlig kortafgift + 299 kr. for modul).

Fødselsdagstilbudet med TV og Boxer gælder så længe lager haves.
